lib/mixlib/install/generator/powershell/scripts/helpers.ps1.erb

```diff
@@ -112,22 +112,13 @@ function Test-ProjectPackage {
   param ($Path, $Algorithm = 'SHA256', $Hash)
 
   if (-not (get-command get-filehash -ErrorAction 'SilentlyContinue')) {
```

Comment:
I think you want to remove this if statement now. I don't think we're using it for anything since we're making a Get-FileHash function below.